lib/install/management/resource_pool.go

```diff
@@ -79,18 +80,25 @@ func (d *Dispatcher) createResourcePool(conf *metadata.VirtualContainerHostConfi
 	return vrp, nil
 }
 
-func (d *Dispatcher) destroyResourcePool(conf *metadata.VirtualContainerHostConfigSpec) error {
-	log.Infof("Destroying the Resource Pool")
+func (d *Dispatcher) destroyResourcePoolIfEmpty(conf *metadata.VirtualContainerHostConfigSpec) error {
+	log.Infof("Removing Resource Pool")
```

Comment:
Let's have the name of the pool as with switch and pg, given it's the outer edge of the VCH from a compute/folder hierarchy perspective.